src/plugins/intel_npu/src/common/CMakeLists.txt

```diff
@@ -14,13 +14,16 @@ set_target_properties(${TARGET_NAME} PROPERTIES EXPORT_NAME npu_common)
 target_include_directories(${TARGET_NAME}
     PUBLIC
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+        $<TARGET_PROPERTY:LevelZero::LevelZero,INTERFACE_INCLUDE_DIRECTORIES>
+        $<TARGET_PROPERTY:LevelZero::NPUExt,INTERFACE_INCLUDE_DIRECTORIES>
 )
 
 target_link_libraries(${TARGET_NAME}
-    PUBLIC
+    PRIVATE
```

Comment:
We use PRIVATE here and export openvino::npu_common.  If I use developer build and depend on openvino::npu_common, have to add deps of openvino::npu_al, levelZero by hand, right? Not sure if use PUBLIC for export targets to propagate reduce more codes on deps

Reply:
done